packages/data-explorer-ui/src/components/Layout/components/Header/components/ProfileComponent/components/authenticationMenu.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { AuthenticationMenu } from "./authenticationMenu";
+
+export default {
+  argTypes: {
+    menuPosition: {
```

Comment:
We probably don't have a need at this time for `menuPosition` as a prop - let's remove it here in the story and in the component for now. We can always add it at a later stage.